src/main/resources/application.properties

```diff
@@ -0,0 +1,24 @@
+spring.application.name=dogBreedApp
+
+#postgre
+spring.r2dbc.url=r2dbc:postgresql://localhost:5432/testDb
+spring.r2dbc.username=user
+spring.r2dbc.password=pass
+
+#flyway
+spring.datasource.url=jdbc:h2:mem:testdb;DB_CLOSE_DELAY=-1;DB_CLOSE_ON_EXIT=FALSE
```

Comment:
Are you using Flyway for tests only?
It must be configured for all datasources, otherwise it won`t make much sense.

Reply:
only for test. T get test data